events/findings/security_finding.json

```diff
@@ -26,6 +26,10 @@
         }
       }
     },
+    "activities": {
```

Comment:
Q: Should we be adding attributes to an object that has already been deprecated?